sync-pingdom-ec2-security-groups.py

```diff
@@ -133,6 +130,11 @@ def main():
         type=int,
         default=80,
         help='The highest port on which Pingdom probes')
+    parser.add_argument(
+        '--rules-per-security-group',
```

Comment:
`README.md` should be updated to include this option.

Reply:
Good catch! Done.

Reply:
Great!